### ISSUE TITLE
Docker Images on Docker Hub are not updating

### ISSUE THREAD

**Issue by zachriggle** [Gallopsled/pwntools#1759]
**TL;DR @heapcrash broke the Docker images by upgrading to Bionic with ddf5cdcb**

## Current Docker Builds

It looks like most images have not updated in 6 months.

<img width="687" alt="Screen Shot 2021-01-12 at 5 29 03 AM" src="https://user-images.githubusercontent.com/111640/104309068-302d4480-5497-11eb-85b1-d99f3110a6bf.png">

## Docker Hub Build Failures

It looks like the builds on Docker Hub may be failing, which could explain this.

Notably, it looks like the `base` image succeeds building, but the follow-up which upgrades Pwntools to `dev` or `beta` are failing.

<img width="1194" alt="Screen Shot 2021-01-12 at 5 30 52 AM" src="https://user-images.githubusercontent.com/111640/104309191-65399700-5497-11eb-8302-2649e65aea4d.png">
<img width="1183" alt="Screen Shot 2021-01-12 at 5 31 01 AM" src="https://user-images.githubusercontent.com/111640/104309196-679bf100-5497-11eb-87d7-f227987aa225.png">

## Python3 is the culprit?

It looks like the issue is that Python3 doesn't have `pip` installed.  From the [build logs](https://hub.docker.com/repository/registry-1.docker.io/pwntools/pwntools/builds/0309ff1c-cd29-402e-9251-3294304af11c) (for @Arusekk mostly since it's behind a login wall)

```
Step 1/5 : FROM pwntools/pwntools:base
---> fe7ac555687e
Step 2/5 : USER root
---> Running in 5589ec307307
Removing intermediate container 5589ec307307
---> 52abdfcfa913
Step 3/5 : RUN pip3 install --upgrade git+https://github.com/Gallopsled/pwntools@stable
---> Running in 1077d26ce736
[91m/bin/sh: 1: pip3: not found
```

It looks like we don't install Python3 in the base image anymore.

```sh
$ docker pull pwntools/pwntools:base
base: Pulling from pwntools/pwntools
Digest: sha256:4e00069c08234a8b94cce25496e614a6501621fb937f629bca73798bdabb4d04
Status: Image is up to date for pwntools/pwntools:base
docker.io/pwntools/pwntools:base
$ docker run -it pwntools/pwntools:base which python
/usr/bin/python
$ docker run -it pwntools/pwntools:base which python3
<nil>
$ docker run -it pwntools/pwntools:base python --version
Python 2.7.12
```

From looking at the history, it looks like @heapcrash broke it with ddf5cdcb by upgrading form Xenial to Bionic.  The timeline (6-7 months) also fits.

**Comment by zachriggle** [comment]
Update: I've enabled e-mail notifications for Docker Hub build failures, so at least next time this happens it won't take 7 months to notice.

**Comment by heapcrash** [comment]
It looks like `python3` is not getting installed despite being in the big list of things that are installed.

```
$ docker run -it pwntools/pwntools:base sudo apt-get install python3-pip
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following additional packages will be installed:
  dh-python libmpdec2 libpython3-dev libpython3-stdlib libpython3.5 libpython3.5-dev libpython3.5-minimal libpython3.5-stdlib python3 python3-dev python3-minimal python3-pkg-resources python3-setuptools
  python3-wheel python3.5 python3.5-dev python3.5-minimal
Suggested packages:
  python3-doc python3-tk python3-venv python-setuptools-doc python3.5-venv python3.5-doc binfmt-support
The following NEW packages will be installed:
  dh-python libmpdec2 libpython3-dev libpython3-stdlib libpython3.5 libpython3.5-dev libpython3.5-minimal libpython3.5-stdlib python3 python3-dev python3-minimal python3-pip python3-pkg-resources
  python3-setuptools python3-wheel python3.5 python3.5-dev python3.5-minimal
0 upgraded, 18 newly installed, 0 to remove and 0 not upgraded.
Need to get 44.0 MB of archives.
After this operation, 84.4 MB of additional disk space will be used.
```

**Comment by heapcrash** [comment]
This is now fixed, Docker Hub was pointing at the wrong Dockerfile (xenial instead of base).

Closed via #1768 and changes on Docker Hub.

```
$ docker run -it pwntools/pwntools:base pwn version                 
[*] Pwntools v4.3.1
```
